woocommerce/woocommerce
# bump wp-env to 9.7.0, include blocks in syncpack

### Changes proposed in this Pull Request:

This PR 
- updates the pinned `@wordpress/env` version to `^9.7.0` in Syncpack
- removes `@wordpress/env` from the excluded packages for the blocks package in Syncpack

Closes #43957 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. In `trunk` run `pnpm list --recursive @wordpress/env` in the root of the repo
2. All packages should be using `^9.7.0`
3. Repeat the above in this branch
4. Run `syncpack list-mismatches`
5. This should not list any packages 
6. 



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 
bump wp-env to 9.7.0, include blocks in syncpack
</details>
